quickstart/azure/setup.md

```diff
@@ -92,17 +94,31 @@ For example, a common Service Principal as displayed by the Azure CLI looks some
 }
 ```
 
+However, you also need to obtain subscription id. To get current subscription Id you can use:
```

Comment:
Consistent capitalization, how about:

"You also need to obtain a Subscription ID. To retrieve your current Subscription ID, you can use ..."

"To list all available subscriptions, you can use..."

Reply:
sounds reasonable.

Reply:
just to be clear, should I change all "subscription id" to "Subscription ID" and leave subscription intact, or should I capitalize "subscription" as well (there are several places where we talk about "subscription", not "Subscription ID")

Reply:
For clarity, I've added Suggested changes with the specific changes that Sean suggested. I don't think there's any other places that would need to change.